### DISCUSSION TITLE
UI flickering

### DISCUSSION THREAD

**Discussion by Suinollat** [ppy/osu#20256] [Q&A] [answered]
### Type

Game behaviour

### Bug description

I had done recordings with OBS, (with both low 6000kbps and lossless quality) but before then I had not had this issue. The issue persists after recording.

### Screenshots or videos

https://user-images.githubusercontent.com/85840855/189591501-8670f479-425e-41fb-9b34-2f8ca759b04a.mp4

### Version

2022.911.0-lazer

### Logs

[performance-draw.log](https://github.com/ppy/osu/files/9545463/performance-draw.log)
[performance-input.log](https://github.com/ppy/osu/files/9545464/performance-input.log)
[performance-update.log](https://github.com/ppy/osu/files/9545465/performance-update.log)
[runtime.log](https://github.com/ppy/osu/files/9545466/runtime.log)
[updater.log](https://github.com/ppy/osu/files/9545467/updater.log)
[database.log](https://github.com/ppy/osu/files/9545468/database.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/9545469/legacy-ipc.log)
[network.log](https://github.com/ppy/osu/files/9545470/network.log)
[performance.log](https://github.com/ppy/osu/files/9545471/performance.log)
[performance-audio.log](https://github.com/ppy/osu/files/9545472/performance-audio.log)

**Comment by peppy** [comment, marked as answer]
This looks like a driver issue. Please download the latest drivers for your GPU.

**Reply by Suinollat** [reply]
Hm, alright. I had just installed the latest driver. I'll revert back

**Reply by peppy** [reply]
For reference, which driver version was exhibiting this issue? Also, which windows version and GPU are you running?

**Reply by Suinollat** [reply]
OS: Windows 11
GPU: AMD RX VEGA 56
Driver that caused the issue: Adrenalin 22.8.2

It wasn't a WHQL driver, so I should have expected as much.
